pkg/bisect/bisect_test.go

```diff
@@ -36,46 +41,108 @@ func (env *testEnv) BuildKernel(compilerBin, userspaceDir, cmdlineFile, sysctlFi
 	return "", nil
 }
 
+func crashErrors(num int, title string) []error {
+	var errors []error
+	for i := 0; i < num; i++ {
+		errors = append(errors, &instance.CrashError{
+			Report: &report.Report{
+				Title: fmt.Sprintf("crashes at %v", title),
+			},
+		})
+	}
+	return errors
+}
+
+func nilErrors(num int) []error {
+	var errors []error
+	for i := 0; i < num; i++ {
+		errors = append(errors, nil)
+	}
+	return errors
+}
+
 func (env *testEnv) Test(numVMs int, reproSyz, reproOpts, reproC []byte) ([]error, error) {
 	hc, err := env.r.HeadCommit()
 	if err != nil {
 		env.t.Fatal(err)
 	}
-	// For cause bisection, if newer than or equal to 602, it crashes.
-	// -- 602 is the cause commit.
-	// TODO: for fix bisection(check env.fix), if older than 602, it crashes.
-	// -- 602 is the fix commit.
+
+	// Regardless of type of bisection, return that a crash occurs.
+	if env.crashAll {
+		return crashErrors(numVMs, hc.Title), nil
+	}
+
 	val, err := strconv.Atoi(hc.Title)
 	if err != nil {
 		env.t.Fatalf("invalid commit title: %v", val)
 	}
-	if val >= 602 {
-		var errors []error
-		for i := 0; i < numVMs; i++ {
-			errors = append(errors, &instance.CrashError{
-				Report: &report.Report{
-					Title: fmt.Sprintf("crashes at %v", hc.Title),
-				},
-			})
+
+	// Returns values that lead bisection to be inconclusive.
+	// Note that inconclusive results can also be caused by returning conflicting
+	// results, but we do not do that here.
```

Comment:
Returning conflicting results should not lead to inconclusive bisection, if there is at least 1 crash, the bug considered to be present.